README.md

```diff
@@ -1,2 +1,52 @@
 # Hello-Microverse
 A simple Microverse Project seeting a Hello World Repository
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Roodjino Cherilus**
+
+- GitHub: [@githubhandle](https://github.com/roodjinocherilus)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/roodjinocherilu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cherilusjcr/)
+
```

Comment:
[OPTIONAL] You can remove this second author since this is done by one person 😃

Reply:
Will do! Thank you !